server/elasticsearch.ts

```diff
@@ -74,6 +74,48 @@ export default class Elasticsearch {
     }));
   }
 
+  /**
+   * Gets values from a specific field from all indices (except if excluded) for one study.
+   * @param id the identifier of the study.
+   * @param field the name of the field to return values from.
+   * @param excludeIndex optionally exclude results from an index.
+   */
+  async getStudyFieldAllIndices(id: string, field: keyof CMMStudy, excludeIndex?: string) {
+    const query: any = {
+      size: 1000,
```

Comment:
Consider making the size a customisable parameter to avoid unneccessary overhead